### ISSUE TITLE
ControlNeXt sdxl training is converging slower than sdxl controlnet?

### ISSUE THREAD

**Issue by universewill** [dvlab-research/ControlNeXt#36]
I trained on my custom dataset, but controlNeXt sdxl training seems converging slower than sdxl controlnet.
I think the problem is the controlnet model size of contronext is too small

**Comment by Pbihao** [comment]
@Eugeoter

**Comment by Eugeoter** [comment]
> I trained on my custom dataset, but controlNeXt sdxl training seems converging slower than sdxl controlnet. I think the problem is the controlnet model size of contronext is too small

Hi, could you please share the details of your training process? I’m interested in knowing which type of control conditions you used (e.g., depth, canny, etc.), as well as the ControlNet scale, learning rates (for both UNet and ControlNet), batch size, and any other relevant parameters. Thanks!

**Comment by Eugeoter** [comment]
> I trained on my custom dataset, but controlNeXt sdxl training seems converging slower than sdxl controlnet. I think the problem is the controlnet model size of contronext is too small

As a reference, for the Canny model, we set the controlnet_scale_factor to 0.35 (for depth, it's 1.0), the learning rate to 1e-5, the ControlNet learning rate to 1e-4, and a batch size of 16. Typically, convergence begins around 2000 steps and is usually achieved within 5000 steps.

**Comment by universewill** [comment]
Basicly i used the default training parameters of controlnext. My image condition is a image with several colors, each color denote a region. For example , green color region is for human body, red color is for face, blue color is for dog body
